src/goto-programs/goto_contractor.h

```diff
@@ -50,7 +254,7 @@ class vart
 class CspMap
 {
 public:
-  static constexpr int MAX_VAR = 10;
+  static constexpr int MAX_VAR = 26;
```

Comment:
removed

Reply:
You removed it but used the magic number in the code directly. Please, use the MAX_VAR constant (or any other constant name FWIW) instead of magic numbers.

My question is why do we need to limit the max number of constants, doesn't the code work with more than 26 variables?

Reply:
I agree with you. It should not be arbitrary. I am still working on this. Because I can't change the size of the vector, I need to recreate it each time we find a new variable. The ibex documentation did not state any limits on the number of variables in a vector.